src/web/routes/admin/auth/integration.test.ts

```diff
@@ -75,44 +75,6 @@ describe('/admin/auth', () => {
 				.expect({ authorized: true });
 		});
 
```

Comment:
**note:** Add new integration tests, reorder existing ones and resolve test flakiness (caused by a time-bound JWT token).